pipeline/terraform/stack/service_work_batcher.tf

```diff
@@ -26,7 +26,7 @@ module "batcher" {
     queue_url        = module.batcher_queue.url
     output_topic_arn = module.batcher_output_topic.arn
 
-    flush_interval_minutes = 30     // NOTE: this needs to be less than visibility timeout
+    flush_interval_minutes = 45     // NOTE: this needs to be less than visibility timeout
```

Comment:
Need to change the visibility timeout too

Reply:
I think I did? https://github.com/wellcomecollection/catalogue/pull/1201/files#diff-e30e33ef26dc6b5e87bf8d15ffbba4203a316167c2346c3f26bb3e2d8fb06384R5

Reply:
This comment was made before that change :)

Reply:
ah sorry only saw it now